### PR TITLE
Added .apiErrorDetails to RTM Message

### PR DESCRIPTION
**Pull request by jeremiahthomasharris**
- cleanup in RTMMessageHandler
- tweaks to logging format

### CODE REVIEW

**Comment by jeremiahthomasharris** [comment]
build passing locally.

File: FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV4.swift [hunk 1]
```diff
@@ -59,15 +59,19 @@ class RtmMessageHandlerV4: RtmMessageHandlerV2 {
         }
     }
     
-    func handleSdkVersion(_ message: RtmMessage) {
+    // MARK: - Private
+    
+    private func handleSdkVersion(_ message: RtmMessage) {
```
Comment (climbonbelay) [inline FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV4.swift]: Is this covered in our list of breaking changes as part of the `RTMConfig` is no longer public note? Or do we need to call this out separately?

File: FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV5.swift [hunk 2]
```diff
@@ -62,15 +66,17 @@ class RtmMessageHandlerV5: RtmMessageHandlerV4 {
         return messageAction.msgHandlerFor(handlerObject: self)
     }
     
-    func handleIdVerificationRequest(_ message: RtmMessage) {
+    // MARK: - Private
+    
+    private func handleIdVerificationRequest(_ message: RtmMessage) {
```
Comment (climbonbelay) [inline FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV5.swift]: Same question as RTM v4

Comment (jeremiahthomasharris) [reply FitpaySDK/Rtm/RtmMessageHandlers/RtmMessageHandlerV5.swift]: No need, they are going from internal to private so it was already hidden from OEMs.

**Pull request merged**